packaging/nnstreamer.spec

```diff
@@ -673,14 +674,14 @@ export NNSTREAMER_CONVERTERS=${NNSTREAMER_BUILD_ROOT_PATH}/ext/nnstreamer/tensor
 
 %define test_script $(pwd)/packaging/run_unittests_binaries.sh
 
-%if %{with tizen}
-    bash %{test_script} ./tests/tizen_nnfw_runtime/unittest_nnfw_runtime_raw
-    bash %{test_script} ./tests/tizen_sensor/unittest_tizen_sensor
-%endif #if tizen
 %if 0%{?unit_test}
     bash %{test_script} ./tests
     bash %{test_script} ./tests/cpp_methods
     bash %{test_script} ./tests/nnstreamer_filter_extensions_common
+%if %{with tizen}
```

Comment:
Q: When is this flag enabled?

Reply:
According to the PK's opinion and considering our CI, I update the condition as below.
```cpp
%if ( %{with tizen} && "%{?profile}" != "tv" )
```